documentation/src/main/wiki/Knot.md

```diff
@@ -1,5 +1,99 @@
+#Knot
 Knot is a module which defines custom step during [[request routing|KnotRouting]]. It can process custom
```

Comment:
Improvement proposition:

It can process markup [[fragments|Splitter]], invoke [[Adapters|Adapter]] and redirect site visitors depending on Adapter response.